src/ConsoleMode.ps1

```diff
@@ -0,0 +1,103 @@
+# Hack! https://gist.github.com/lzybkr/f2059cb2ee8d0c13c65ab933b75e998c
+
+if ($IsWindows -eq $false) {
```

Comment:
Thought is that `$IsWindows` will be `$null` for PS `-lt 6`, and `$null -ne $false`. Net result is that we'll only create `NativeConsoleMethods` if we're not explicitly on not-Windows.

Or are you thinking that we shouldn't create `NativeConsoleMethods` for older versions of PS?